src/Symfony/Component/PropertyAccess/composer.json

```diff
@@ -17,7 +17,8 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "symfony/polyfill-php70": "~1.0"
+        "symfony/polyfill-php70": "~1.0",
+        "doctrine/annotations": "~1.2"
```

Comment:
I don't think that's a good idea to add a new dependency to this component.
Maybe put it in ``required-dev`` / ``suggest`` instead ?